src/token/memberlist.sol

```diff
@@ -28,7 +28,7 @@ contract Memberlist {
     }
 
     function updateMember(address usr, uint validUntil) public auth {
-        require((safeAdd(block.timestamp, minimumDelay)) < validUntil, "invalid-validUntil");
+        require((safeAdd(block.timestamp, minimumDelay)) < validUntil, "cent/invalid-validUntil");
```

Comment:
Why this change?

Reply:
yeh, in fact a lot of other errors are prefixed with `CentrifugeConnector/`

Reply:
I just removed this change to keep these contracts unaltered